packages/taste-test/src/lib/contants.ts

```diff
@@ -0,0 +1,24 @@
+import { fromText } from "lucid-cardano";
+
+export const SETNODE_PREFIX = "FSN";
+export const CORRNODE_PREFIX = "FCN";
+export const CFOLD = "CFold";
+export const PTHOLDER = "PTHolder";
+export const TWENTY_FOUR_HOURS_MS = 86_400_000;
+export const ONE_HOUR_MS = 3_600_000;
+
+export const originNodeTokenName = fromText(SETNODE_PREFIX);
+export const corrNodeTokenName = fromText(CORRNODE_PREFIX);
+export const cFold = fromText(CFOLD);
+
+export const NODE_ADA = 3_000_000n;
```

Comment:
What does "NODE_ADA" mean? What do the 3 ADA stand for here?

Reply:
Think of it like the minUTXO deposit we have on the sundaeswap contracts.